app/_locales/zh_TW/messages.json

```diff
@@ -235,7 +235,7 @@
   "done": {
     "message": "完成"
   },
-  "downloadStatelogs": {
+  "downloadStateLogs": {
```

Comment:
Did you also change it in the template, or was it already different there?

Reply:
theses are the stragglers, the rest were already changed in locales and templates